stac/education/bloom/collection.json

```diff
@@ -26,7 +29,6 @@
         }
     ],
     "stac_extensions": [],
-    "pl:item_type_id": "PSScene",
```

Comment:
What is this for? Removed for now because it's not defined in the Planet STAC extension.